atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/KvsMigrationCommand.java

```diff
@@ -184,6 +231,14 @@ private AtlasDbConfig overrideTransactionTimeoutMillis(AtlasDbConfig config) {
                 .build();
     }
 
+    private AtlasDbRuntimeConfig disableSweep(AtlasDbRuntimeConfig atlasDbRuntimeConfig) {
+        return ImmutableAtlasDbRuntimeConfig.builder()
+                .from(atlasDbRuntimeConfig)
+                .sweep(SweepConfig.disabled())
+                .targetedSweep(TargetedSweepRuntimeConfig.disabled())
+                .build();
```

Comment:
This happened to serendipitously work... but we really should turn off the compaction configs too as we don't want the CLI to do that even if e.g. large internal product overrides it.

Reply:
Ack, thanks. Have added an explicit disabled() method on compactor config (while it's disabled by default, all it would take is a change of the defaults for it to break CLI). I debated inlining it but went with an extra method for consistency.